backends/platform/sdl/sdl-sys.h

```diff
@@ -111,6 +111,32 @@
 
 #endif // FORBIDDEN_SYMBOL_EXCEPTION_time_h
 
+// Fix compilation on armv7hl, aarch64, ppc64le and s390x architecture
+// It needs various (usually forbidden) symbols from time.h
```

Comment:
There is a bad copy paste here.
Could you please replace time.h with unistd.h?
In the same time could you edit the comment above? It will fail on all non-x86 platforms.